core/src/main/java/org/apache/accumulo/core/conf/LogSync.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.conf;
+
+/**
+ * Constants to be used for the DatafileTransactionLog
+ */
+public enum LogSync {
+  log, logsync, metasync, memsync
```

Comment:
Convention is to make the constants all capitals: `LOG, LOGSYNC, ...`

Reply:
but the convension is lowercase for property values....

Reply:
I will comment as to why they are lowercase.

Reply:
Yeah, what i've usually done in those scenarios are make a label attribute or something in the enum: `LOG("log"), LOGSYNC("logsync")...`. I'd lean towards that personally just to align with java convention.

Reply:
I'm not sure this enum is necessary. I think it could be removed.

Reply:
If not an enum then I would like to create constants somewhere.  Do we have a preference where those might live?  In the PropertyType class itself?